packages/vue/vue-vanilla/src/array/ArrayListRenderer.vue

```diff
@@ -4,6 +4,7 @@
       <button
         :class="styles.arrayList.addButton"
         type="button"
+        :disabled="!control.enabled || maxItemsReached"
```

Comment:
good catch to also check `control.enabled`. Can you also add the check to all the other buttons?